lib/rules/no-useless-catch.js

```diff
@@ -43,11 +45,31 @@ module.exports = {
 						context.report({
 							node,
 							messageId: "unnecessaryCatchClause",
+							fix: fixer => fixer.remove(node),
 						});
 					} else {
 						context.report({
 							node: node.parent,
 							messageId: "unnecessaryCatch",
+							fix(fixer) {
+								const tryBlock = node.parent.block;
+								const sourceCode = context.getSourceCode();
```

Comment:
```suggestion
								const sourceCode = context.sourceCode;
```

`context.getSourceCode()` is deprecated.